themes/docdock/static/theme-flex/style.css

```diff
@@ -714,6 +714,69 @@ div.notices.help {
   color: rgba(132, 56, 56, 0.995) !important;
 }
 
+.panel {
+  background-color: white !important; }
+  .panel-body {
+    color: #323232 !important; }
+
+/* This feels so wrong... I'm re-theming bootstrap in here, there has to be a cleaner way. */
+/* TODO: Build bootstrap with the rest of the CSS? */
```

Comment:
You're right, this is a bit weird but I'm definitely not skilled enough in CSS to understand how this will work.

Reply:
oh sorry, this was a copy of something Dan wrote :P

Reply:
Haha no worries.